src/replaceAroundStep.ts

```diff
@@ -25,6 +28,83 @@ export function suggestReplaceAroundStep(
   prevSteps: Step[],
   suggestionId: number,
 ) {
+  // This detects and handles changes from `setNodeMarkup`
+  // (https://github.com/handlewithcarecollective/prosemirror-suggest-changes/issues/7)
+  if (
+    step.insert === 1 &&
+    step.from === 0 &&
+    step.slice.size === 2 &&
+    step.gapTo === step.to - 1 &&
+    step.gapFrom === step.from + 1 &&
+    (step as any).structure === true
```

Comment:
Huh... I hadn't realized that `structure` was internal, that kind of stinks. Can we cast this to `ReplaceAroundStep & { structure: boolean }` instead, to be a bit more precise?